devlab/tests/test_group_verification.py

```diff
@@ -34,9 +34,11 @@ def setUp(self):
         #  only way at this moment.
         #  Should be fixed by implementing proper package module for Cloud
         #  Ferry.
+        self.conf_path = 'devlab/tests/'
         cmd_no_path = './../../devlab/provision/generate_config.sh' \
                       ' --cloudferry-path '
-        cmd = os.path.join(cmd_no_path, self.main_folder.lstrip('/'))
+        cmd = os.path.join(cmd_no_path, self.main_folder.lstrip('/'),
+                           ' ' + str(self.main_folder), self.conf_path.rstrip('/'))
```

Comment:
You don't need to care about slashes if you use `os.path.join`, this function handles this for you, please remove all the `[r|l]strip()` calls.